src/https/utilities/https-utilities

```diff
@@ -70,6 +72,8 @@ run_certbot()
 	certbot --text --config-dir $CERTBOT_DIRECTORY/config \
 	               --work-dir $CERTBOT_DIRECTORY/work \
 	               --logs-dir $CERTBOT_DIRECTORY/logs $@
+
+	openssl ecparam -name secp384r1 >> $LIVE_CERT
```

Comment:
>SSLOpenSSLConfCmd DHParameters is enough. Can you verify?

That's the theory, but it didn't work in my env. Feel free to prove me wrong :) and btw, it's another command which needs to be used for curves: `ECDHParameters`.

Also, Apache recommends doing this via adding the parameter to the cert. It may be more reliable.